sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,63 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
https://github.com/OCA/stock-logistics-transport/blob/8.0/stock_shipment_management/README.rst
https://www.odoo.com/apps/modules/8.0/stock_shipment_management/
I added a logo after title and one before title using http instead of https

And in odoo it is well displayed. Don't they use the same lib in odoo and on apps?

I sent a message to apps@odoo.com the issue is clearly on their side